LICENSE

```diff
@@ -1,27 +1,25 @@
-Copyright (c) Django Software Foundation and individual contributors.
+Copyright (c) 2001-2017 Django Software Foundation and individual contributors.
```

Comment:
Is it required to include the date? I find that annoying to update each year, though we already have to do it for the Python license.

Reply:
The answer is no. It is not required to make GitHub recognize the license.
- It is however included in the default BSD-3-Clause license text.

Reply:
I have removed the year from the License.